ios/FluentUI/Notification/NotificationTokens.swift

```diff
@@ -168,15 +168,15 @@ open class NotificationTokens: ControlTokens {
     }
 
     /// The value for the bottom padding between the notification and its anchor view
-    open var bottomPresentationPadding: CGFloat { 20.0 }
+    open var bottomPresentationPadding: CGFloat { globalTokens.spacing[.medium] }
```

Comment:
is it meant to be 16? or is it meant to be 20?

Reply:
16!

<img width="586" alt="image" src="https://user-images.githubusercontent.com/31874971/184423204-9cb41b7e-5449-41c7-8dec-8759ec27867f.png">